book/Zapp-Pipes/5.-Feed-API.md

```diff
@@ -214,14 +214,24 @@ The media_item element MUST be contained inside a  media_group element.
 - Keys are of String type
 - Values are of ANY type
 
-##### expires_at
+#### expires_at
 
 This field, under the extensions object, defines the time in which the component that holds this Feed as a datasource needs to be refreshed. For example, if your Feed contains program which ends in 30 minutes and you want to remove this program from the screen after it's over - the provider can define the `expires_at` value to be now + 30 minutes (date formatted string with this format `"yyyy/MM/dd HH:mm:ss Z"`). Then the component will trigger the provider's load method again when the time comes.
 
 #### free
 
 The `free` key can be defined on the extensions object to indicate whether an item is free or not. The value can either be a String representation of a Boolean (true/false) or a Boolean. If the extension key is not defined or a bad value was supplied the default item behavior will be as free.
 
+#### video_ads
+The datasource can provide information about video ads to display with the video item.
+The video ads configuration should then be handled by the Video Player to display ads before, after or during the video playback.
```

Comment:
Lets change it from `configuration should then be handled` to `configuration will be handled by the video player if it supports displaying video ads`